bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
+
+MAIL_TO = ["fsfod11@gmail.com"]
+
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop apt-daily.timer",
+    "sudo systemctl --runtime disable apt-daily.timer", 
+    "sudo systemctl stop apt-daily-upgrade.timer",
+    "sudo systemctl --runtime disable apt-daily-upgrade.timer",
+    
+    "sudo systemctl stop systemd-timesyncd",
+    "sudo systemctl --runtime disable systemd-timesyncd",
+    "sudo systemctl stop cron",
+]
+
+POST_EXECUTION_CMDS = [
+    "sudo systemctl start systemd-timesyncd || true",
+    "sudo systemctl start cron || true",
```

Comment:
You need to bring up everything you turned off here.

Reply:
Doe krun stop looking at dmesg when these are run?

Reply:
krun only looks at the dmesg during the process execution.

Reply:
well i was trying to figure out if that was the problem before